cfgov/jobmanager/models/pages.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
```

Comment:
Are there still definitions inside `jobmanager/models.py` ?   The hierarchy you're presenting makes sense to me, but I feel it should be consistent